src/main.js

```diff
@@ -1,19 +1,16 @@
+// @flow
 /**
  * The main entrypoint for our react-component render server.
  */
 
-"use strict";
+import express from "express";
+import logging, {middleware} from "./logging.js";
 
 // Now that cloud trace is set up, we can require() everything else.
```

Comment:
I think this comment is no longer accurate?

Reply:
Good point!